assets/js/mbta.ts

```diff
@@ -1987,16 +1987,65 @@ const stationConfig: {
   Green: {
     batchModes: { auto: true, headway: true, off: true },
     stations: [
+      {
+        id: 'GUNS',
+        name: 'Union Sq',
+        zones: {
+          e: {
+            label: 'Track 1',
+            modes: {
+              auto: false,
+              custom: true,
+              headway: false,
+              off: true,
+            },
+          },
+          w: {
+            label: 'Track 2',
+            modes: {
+              auto: false,
+              custom: true,
+              headway: false,
+              off: true,
+            },
+          },
+          m: {
+            modes: {
+              auto: false,
+              custom: true,
+              headway: false,
+              off: true,
+            },
+          },
+        },
+      },
       {
         id: 'GLEC',
         name: 'Lechmere',
         zones: {
+          e: {
+            label: 'Union Sq/College Ave',
```

Comment:
Does this fit into the UI? We might want something shorter. (I assume it's fine if it's `Union Sq` for now, until the rest of GLX opens)